modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -427,6 +427,7 @@ def transform_disabilities # rubocop:disable Metrics/MethodLength
           disability.delete('diagnosticCode')
           disability.delete('disabilityActionType')
           disability.delete('isRelatedToToxicExposure')
+          disability.delete('specialIssues')
           sec_dis = disability['secondaryDisabilities']&.map do |secondary_disability|
```

Comment:
@mchristiansonVA Do we need to also delete the secondary disability special issues?